src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverProjectExtension.kt

```diff
@@ -232,7 +252,26 @@ internal interface KoverMppSourceSet {
     public fun compilation(compilationName: String)
 }
 
-
+/**
+ * Exclude classes from instrumentation.
```

Comment:
Here and everywhere, opening verbs in KDoc are in a third-person singular form.
Exclude -> Excludes etc.